src/utils/HealthLogger.js

```diff
@@ -165,6 +174,92 @@ define(function (require, exports, module) {
             fileEncCountMap[encoding]++;
             setHealthData(healthData);
         }
+
+
+        sendAnalyticsData(commonStrings.USAGE + commonStrings.FILE_OPEN + language._name,
+                            commonStrings.USAGE,
+                            commonStrings.FILE_OPEN,
+                            language._name.toLowerCase()
+                         );
+
+    }
+
+    /**
+     * Whenever a file is saved call this function.
+     * The function will send the analytics Data
+     * We only log the standard filetypes and fileSize
+     * @param {String} filePath The path of the file to be registered
+     */
+    function fileSaved(docToSave) {
+        if (!docToSave) {
+            return;
+        }
+        var fileType = docToSave.language ? docToSave.language._name : "";
+        sendAnalyticsData(commonStrings.USAGE + commonStrings.FILE_SAVE + fileType,
+                            commonStrings.USAGE,
+                            commonStrings.FILE_SAVE,
+                            fileType.toLowerCase()
+                         );
+    }
+
+    /**
+     * Whenever a file is closed call this function.
+     * The function will send the analytics Data.
+     * We only log the standard filetypes and fileSize
+     * @param {String} filePath The path of the file to be registered
+     */
+    function fileClosed(file) {
+        if (!file) {
+            return;
+        }
+        var language = LanguageManager.getLanguageForPath(file._path),
+            size = -1;
+
+        function _sendData(fileSize) {
+            var subType = "";
+
+            if(fileSize/1024 < 1) {
+
+                if(fileSize === -1) {
+                    subType = "";
+                }
+                if(fileSize < 10) {
```

Comment:
shouldn't the check be <=? if the file size is 10kb, it will log in 10_50KB range.

Reply:
fixed